blob/blob.go

```diff
@@ -2,12 +2,17 @@ package blob
 
 import (
 	"bytes"
+	"fmt"
 
+	appns "github.com/celestiaorg/celestia-app/pkg/namespace"
 	"github.com/celestiaorg/celestia-app/x/blob/types"
 	"github.com/celestiaorg/nmt"
 	"github.com/celestiaorg/nmt/namespace"
 )
 
+// versionIndex defines the position of the namespace version in the namespace
```

Comment:
```suggestion
// versionIndex defines the position of the namespace version in the namespaceID
```

Reply:
lets call it `namespaceVersionIndex`

Reply:
preference to call it `namespace` here and not `namespaceID` because this `namespace` contains the version and ID. See https://github.com/celestiaorg/celestia-node/pull/2269#discussion_r1209406533

Reply:
The only thing is that this got removed